pegasus/sites.v3/code.org/public/learn/index.haml

```diff
@@ -122,7 +125,8 @@ style_min: false
       tutorials: #{tutorials_contents.to_json},
       locale: "#{request.locale}",
       roboticsButtonUrl: "/learn/robotics",
-      disabledTutorials: #{DCDO.get('learn_hide_tutorials', [])}
+      disabledTutorials: #{sort_by_popularity},
```

Comment:
How come you removed the DCDO? We can't default by popularity and still have the ability to hide a few (although unlikely to happen again until next HoC)?

Reply:
Ooh, bad typo.  Let me fix.